docs/specs/user-model.md

```diff
@@ -261,18 +261,25 @@ authenticator:
 
 The OTP message is rendered by a [customizable template](./templates.md#otp_message).
 
-### Bearer Token Authenticator
+### Device Token
 
-Bearer token authenticator is secondary.
+Device tokens are used to indicate a trusted device.
 
-A bearer token is generated during MFA when the user opt to skip MFA next time.
+A device token is generated when user opts in during secondary authentication.
+The generated device token is stored in a cookie, and it allows the user to skip
+secondary authentication as long as it remains valid.
 
-The token is a cryptographically secure random string of 256 bits (32 bytes) in hex encoding.
+The token is a cryptographically secure random string with at least 256 bits.
 
-### Recovery Code Authenticator
+### Recovery Code
 
-Recovery code authenticator is secondary.
+Recovery codes are used to skip secondary authentication when a secondary
```

Comment:
It is not accurate to say recovery code is used to skip secondary authentication.